roles/contiv_network/tasks/ovs.yml

```diff
@@ -52,3 +56,11 @@
   with_items:
     - "tcp:127.0.0.1:6640"
     - "ptcp:6640"
+
+- name: disable SE linux
```

Comment:
Why is this needed? It goes against what is recommended by the distribution and it seems a bad idea in the context of running things in Docker.